src/pages/developers/tbtc-security-model.md

```diff
@@ -0,0 +1,126 @@
+---
+template: resource
+title: tBTC Security Model
+heading: tBTC Security Model
+description: >-
+  The following is an overview of tBTC’s security model and how it achieves a
+  high level of security for users and signers.
+date: 2020-04-18T18:21:28.438Z
+tags:
+  - security
+---
+tBTC is designed as the safe way for people to use Bitcoin on Ethereum. Different projects take differing approaches to security. The following is an overview of tBTC’s security model and how it achieves a high level of security for users and signers.
+
+## A trust-minimized Bitcoin sidechain
+
+Sidechains were invented as a way to experiment with unlocking Bitcoin’s potential on other chains. The idea is simple - lock BTC on the main Bitcoin blockchain, and unlock it for use on another blockchain, creating a new currency pegged to BTC. In practice, building a 2-way peg without trusted intermediaries is difficult. Blockstream’s Liquid is an example of a Bitcoin sidechain in production that relies on trusted intermediaries.
+
+Although Liquid works, it lacks BTC’s openness and security. Liquid relies on a trusted federation of signers to custody the “locked” bitcoin. If the signers so desire, they can censor withdrawals or walk away with funds. Further, there is no user recourse in the event of signer malice - users must rely completely on trusting the Liquid network and its underlying technology.
+
+tBTC is a **trust-minimized** alternative to this model. Signers are overcollateralized to 150% the value of the deposit they custody, and users are fully reimbursed in the event of signer fraud. Signers are chosen from an open and decentralized network of nodes that run the Keep protocol, all bonded on Ethereum.
+
+The basic operation of minting 1 TBTC, the pegged version of Bitcoin on Ethereum, is as follows:
+
+1. A bonded ECDSA keep is requested from the Keep network to sign for 1 BTC.
+2. A **signer group** forms and puts down **150% the value of 1 BTC in ETH** as collateral.
```

Comment:
how is it put down as collateral? is that a smart contract call that transfers the signers ETH somewhere that the cant access?

Reply:
The keep contract backing a tBTC deposit acts as an escrow for all signers. It can seize the funds if they're malicious, and run the auction if it goes into liquidation.